specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -4335,6 +4335,64 @@
       "required": [
         "functionAppUrl"
       ]
+    },
+    "SapOpenHubLinkedService": {
+      "x-ms-discriminator-value": "SapOpenHub",
+      "description": "SAP Business Warehouse Open Hub Destination Linked Service.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/LinkedService"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "description": "Properties specific to this linked service type.",
```

Comment:
Please add a more specific description

Reply:
Hi @antmarti-microsoft thanks very much for your comments. I refined this accordingly and merge the change in https://github.com/Azure/azure-rest-api-specs/pull/5195. could you please take a look? Thanks.

Reply:
Will abandon this if that gets merged thanks.